packages/wonder-blocks-dropdown/src/components/dropdown-core.tsx

```diff
@@ -571,7 +571,9 @@ class DropdownCore extends React.Component<Props, State> {
     }
 
     focusPreviousItem(): void {
-        if (this.focusedIndex === 0) {
+        if (this.isSearchFieldFocused() && !this.props.enableTypeAhead) {
+            this.focusedIndex = this.state.itemRefs.length - 1;
+        } else if (this.focusedIndex === 0) {
```

Comment:
**suggestion:** I think this could be moved to the original `if`, so we can simplify assigning `focusedIndex` to this condition:

```suggestion
        if (this.focusedIndex === 0 || (this.isSearchFieldFocused() && !this.props.enableTypeAhead)) {
```